gulpfile.js

```diff
@@ -182,12 +188,23 @@ const setupCordova = gulp.series(cordovaPlatformAdd, cordovaPrepare, xcode);
 //////////////////
 //////////////////
 
+function preFlightValidation(cb) {
+  log.info(os.platform());
+  if (os.platform() !== 'darwin' && platform === 'ios') {
+    log.error(
+        '\x1b[31m%s\x1b[0m',  // Red text
+        'Building the ios client requires xcodebuild and can only be done on MacOS');
+    process.exit(1);
+  }
+  cb();
```

Comment:
Is the callback necessary?

Reply:
Yes, otherwise the piping fails.  It's the same as `done`() in our unit tests, all the tasks in gulp are async.